account-kit/react/package.json

```diff
@@ -37,6 +37,7 @@
   "scripts": {
     "prebuild": "npx tsx ./inject-version.ts",
     "dev": "storybook dev",
+    "dev:build": "lerna watch --scope='@account-kit/react' --include-dependencies -- lerna run build --scope=@account-kit/react",
```

Comment:
what cases does this cover over the `yarn dev` script? I remember testing this some before and I didn't have to build when I made changes to files in `@account-kit/react` because storybook was using the local TS files instead of the built output of the lib

Reply:
also nit, if we do need the script can we make it `build:dev`?

Reply:
I wrote a more detailed description in the channel, should have posted it here as well my bad. 

I found that I was missing regressions when switching between branches because I forgot to run build on our account kit library resulting in the wrong version rendering in the demo app locally.  This script dev:build will alleviate that issue by watching changes in the account-kit/react package and rebuild when necessary.

Reply:
should we just include this in the `dev` script then? can we have it run `storybook dev` and the `lerna` script?

in the past i've used `stmux` to start these watcher tasks in split windows. Something like this:
```
"dev": "stmux -w always -M -- [ -t storybook \"storybook dev\" .. -t build \"yarn build:dev\" ]"
```

Reply:
IMO we should keep them separate as it will significantly slow down build speeds if we only intend to be working in storybook